vault/identity_store_entities_test.go

```diff
@@ -407,6 +407,58 @@ func TestIdentityStore_EntityCreateUpdate(t *testing.T) {
 	}
 }
 
+func TestIdentityStore_BatchDelete(t *testing.T) {
+	ctx := namespace.RootContext(nil)
+	is, _, _ := testIdentityStoreWithGithubAuth(ctx, t)
+
+	ids := make([]string, 10000)
```

Comment:
Same comment here as well. We might want to create more entries than what is being deleted to be sure that we are not removing more than what is necessary.